pyblish_qml/control.py

```diff
@@ -617,6 +617,11 @@ def echo(self, data):
         """Append `data` to result model"""
         self.data["models"]["result"].add_item(data)
 
+    def comment_sync(self, comment):
+        """Update comments to host and notify subscribers"""
+        self.host.update(key="comment", value=comment)
+        self.host.emit("commented", comment=comment)
+
```

Comment:
I was trying this kind of simple (lazy) fix.
Dose this work for you ? @mottosso